<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TowerEditor</class>
 <widget class="QWidget" name="TowerEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tower Editor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="leftLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Existing Towers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="towerList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addTowerButton">
        <property name="text">
         <string>+ Add New Tower</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="rightLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="towerPreview">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="styleSheet">
          <string>border: 2px dashed #888; background: #333; color: #aaa;</string>
         </property>
         <property name="text">
          <string>No preview</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chooseTextureButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Choose Texture...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListWidget" name="nextTowerList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="addNextTowerButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Add next tower</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeNextTowerButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Remove next tower</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>774</width>
          <height>307</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="editorLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QFormLayout" name="propertiesForm">
           <property name="labelAlignment">
            <set>Qt::AlignmentFlag::AlignRight</set>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonLayout">
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="styleSheet">
          <string>color: red;</string>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
